backend/src/main/java/com/redhat/cloud/notifications/templates/Rhosak.java

```diff
@@ -0,0 +1,75 @@
+package com.redhat.cloud.notifications.templates;
+
+import com.redhat.cloud.notifications.models.EmailSubscriptionType;
+import io.quarkus.qute.CheckedTemplate;
+import io.quarkus.qute.TemplateInstance;
+
+// Name needs to be "Rhosak" to read templates from resources/templates/Rhosak
+public class Rhosak implements EmailTemplate {
+    private static final String SCHEDULED_UPGRADE = "scheduled_upgrade";
+    private static final String DISRUPTION = "disruption";
+    private static final String INSTANCE_CREATED = "instance_created";
+    private static final String INSTANCE_DELETED = "instance_deleted";
+    private static final String ACTION_REQUIRED = "action_required";
+
+    @Override
+    public TemplateInstance getTitle(String eventType, EmailSubscriptionType type) {
+        if (eventType.equals(SCHEDULED_UPGRADE)) {
+            return Templates.scheduledUpgradeTitle();
+        } else if (eventType.equals(DISRUPTION)) {
+            return Templates.serviceDisruptionTitle();
+        } else if (eventType.equals(INSTANCE_CREATED)) {
+            return Templates.instanceCreatedTitle();
+        } else if (eventType.equals(INSTANCE_DELETED)) {
+            return Templates.instanceDeletedTitle();
+        } else if (eventType.equals(ACTION_REQUIRED)) {
+            return Templates.actionRequiredTitle();
+        }
+        return null;
+    }
+
+    @Override
+    public TemplateInstance getBody(String eventType, EmailSubscriptionType type) {
+        if (eventType.equals(SCHEDULED_UPGRADE)) {
+            return Templates.scheduledUpgradeBody();
+        } else if (eventType.equals(DISRUPTION)) {
+            return Templates.serviceDisruptionBody();
+        } else if (eventType.equals(INSTANCE_CREATED)) {
+            return Templates.instanceCreatedBody();
+        } else if (eventType.equals(INSTANCE_DELETED)) {
+            return Templates.instanceDeletedBody();
+        } else if (eventType.equals(ACTION_REQUIRED)) {
+            return Templates.actionRequiredBody();
+        }
+        return null;
```

Comment:
@josejulio Same question here.

Reply:
I've changed it to throw an exception instead for consistency.